PocketKit/Sources/PocketKit/MyList/MyListContainerViewController.swift

```diff
@@ -46,6 +48,21 @@ class MyListContainerViewController: UIViewController {
         fatalError()
     }
     
+    private func resetTitleView() {
```

Comment:
I remember why we do this - to reset the title view to have "My List" selected, but I'm wondering if it's worth going into the title view and adding something similar to the `selectedIndex` property. Non-blocker.

Reply:
discussed that this will be addressed if needed in the future and not part of this work